semgrep-core/src/matching/Matching_generic.mli

```diff
@@ -181,11 +180,7 @@ val m_string_prefix : string -> string -> tin -> tout
 
 (*e: signature [[Matching_generic.m_string_prefix]] *)
 val m_string_ellipsis_or_regexp_or_default :
-  ?m_string_for_default:(string, string) matcher ->
-  string ->
-  string ->
-  tin ->
-  tout
+  ?m_string_for_default:string matcher -> string -> string -> tin -> tout
```

Comment:
Could this be `string matcher -> string matcher` ?

Reply:
yep